spec/factories/factories.rb

```diff
@@ -3,12 +3,13 @@
 FactoryGirl.define do
 
   factory :user do |u|
-    u.name                   "Test User that loves beer"
-    sequence(:email) { |i| "user_#{i}@example.com"}
-    u.password               "draft1"
-    u.password_confirmation  "draft1"
-    u.subscribe_lesson_notifications   true
-    u.admin                   false
+    u.name                            "Test User that loves beer"
+    sequence(:email)                  { |i| "user_#{i}@example.com"}
+    u.password                        "draft1"
+    u.password_confirmation           "draft1"
+    u.subscribe_lesson_notifications  true
+    u.subscribe_badge_notifications   true
+    u.admin                           false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.